node_modules/oae-core/signin/css/signin.css

```diff
@@ -0,0 +1,61 @@
+/*!
+ * Copyright 2015 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#signin-modal {
+    text-align: center;
```

Comment:
All the contents in this modal are structurally centered (the external auth buttons with the horizontal "or" ruler with the username/password form). The other modals in the app don't have this kind of structure. Might be better to move it down to the body though